eclipse/build.gradle.kts

```diff
@@ -58,4 +63,131 @@ tasks {
     artifacts {
         add("testing", testJar)
     }
+
+
+    /* Eclipse release tasks
+     *
+     * The following tasks provide the functionality of creating
+     * an eclipse update-site (via "updateSite") or dropin (via "dropin").
+     * The creation of the update-site uses as recommended the eclipse's
+     * P2Directory. You can find a how-to here:
+     * http://maksim.sorokin.dk/it/2010/11/26/creating-a-p2-repository-from-features-and-plugins/
+     *
+     * Instead of calling the P2Director via CLI we use the goomph
+     * plugin's classes that provide an abstraction layer of the P2Director.
+     */
+
+    val updateSiteDirPath = "build/update-site"
+
+    /* Step 0 of update-site creation
+     *
+     * Creates the structure:
+     * update-site/
+     *  |- features/
+     *    |- feature.xml
+     *  |- plugins/
+     *    |- saros.core.jar
+     *    |- saros.eclipse.jar
+     *  |- category.xml
+     */
+    val updateSitePreparation by registering(Copy::class) {
+        dependsOn(":saros.core:jar", ":saros.eclipse:jar")
+
+        from("feature/feature.xml") {
+            into("features")
```

Comment:
Why does this work? I.e., why can you just use the path relative to `build/update-site` here while you still seem to have to provide the base path for the `site.xml` (l.103).